.github/workflows/ci.yaml

```diff
@@ -233,8 +233,7 @@ jobs:
   gen:
     timeout-minutes: 8
     runs-on: ${{ github.repository_owner == 'coder' && 'depot-ubuntu-22.04-8' || 'ubuntu-latest' }}
-    needs: changes
-    if: needs.changes.outputs.docs-only == 'true' || needs.changes.outputs.ci == 'true' || github.ref == 'refs/heads/main'
```

Comment:
I think it's worth making it explicit here, it seems quite difficult to find the documentation that says the default condition is `always()` and not `success()`.

Reply:
The default is `success()`, I think. Why do we need `always()`?

Reply:
The old condition didn't check if the previous job had succeeded, so I don't think we need to start checking it now. My understanding is that with `success()` gen CI wouldn't run if lint failed? Which would mean it would wait for `lint` to finish, when they could just be started concurrently. Please correct me if I'm wrong.